### PR TITLE
Next: Refactoring to support Nette v3

### PR DESCRIPTION
**Pull request by mabar**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,13 +12,14 @@
   ],
   "require": {
     "php": "^7.2",
-    "nette/application": "~2.4.13",
-    "nette/di": "~2.4.15",
-    "nette/forms": "~2.4.9"
+    "nette/application": "~3.0.0",
+    "nette/di": "~3.0.0-stable",
```
Comment (f3l1x) [inline composer.json]: Stable?

File: composer.json [hunk 2]
```diff
@@ -12,13 +12,14 @@
...
+    "nette/forms": "~3.0.09"
```
Comment (f3l1x) [inline composer.json]: 09?

File: composer.json (hunk 1 above)
Comment (mabar) [reply composer.json]: It work's same way as conflict with RC version, it is just more reliable.

File: composer.json (hunk 2 above)
Comment (mabar) [reply composer.json]: Typo, fixed

File: composer.json (hunk 1 above)
Comment (f3l1x) [reply composer.json]: Switch it to normal version and set the conflict, like in the others.

**Pull request merged**